example-react-app/src/index.tsx

```diff
@@ -30,7 +33,12 @@ ReactDOM.render(
       antdLocaleMapping={antdLocaleMapping}
     >
       <HashRouter>
-        <Route component={App} />
+        <DevSupport
+          ComponentPreviews={<Route component={ComponentPreviews} />}
+          useInitialHook={useDevLogin}
```

Comment:
maybe just `initHook`?

Reply:
I set this prop name to **useInitialHook** on purpose, so that prop is used according to the hooks rules.